SConstruct

```diff
@@ -103,8 +103,7 @@ Help(opts.GenerateHelpText(env))
 
 env.Append( CCFLAGS = [ '-DPACKAGE_VERSION=\\"' + NPVERSION + '\\"' ] )
 
-if env['datadir'] != '':
```

Comment:
why remove this check?
also I think it should be `if 'datadir' in env`

Reply:
Not needed. More conditions would just be needed elsewhere in the cpp code. I don't remember exactly why it was causing me problems though, tbh. If it is defined, then DATADIR is defined as NP_DATADIR, so it doesn't break anything. (package.cpp:L30)

Reply:
Verified not needed